include/CLI/App.hpp

```diff
@@ -1176,7 +1198,7 @@ class App {
     }
     /// Get a pointer to subcommand by index
     App *get_subcommand(int index = 0) const {
-        if((index >= 0) && (index < subcommands_.size()))
+        if((index >= 0) && ((size_t)index < subcommands_.size()))
```

Comment:
I suppose really you could use static cast and get rid of the (index>=0) check since a negative number cast into a large unsigned number.  I usually go the other way and cast the sizes to an int but It doesn't make that much difference.

Reply:
In this case, since the index was already checked for `>= 0`, this should be valid. This should be valid for all sizes of ints. Though no one is likely to have something large enough to matter.